packages/portal-proto/src/features/facets/NumericRangeFacet.tsx

```diff
@@ -741,7 +751,7 @@ const Year: React.FC<NumericFacetData> = ({
     <div className="flex flex-col w-100 space-y-2 px-2 mt-1 ">
       <RangeInputWithPrefixedRanges
         hooks={{ ...hooks }}
-        units="year"
+        units="years"
```

Comment:
Note: There is a year type, which is a calendar year, vs. years, which are actually filtered by days.